src/main/java/org/mule/service/http/impl/service/client/async/ResponseBodyDeferringAsyncHandler.java

```diff
@@ -147,6 +153,7 @@ public STATE onBodyPartReceived(HttpResponseBodyPart bodyPart) throws Exception
     if (!input.isPresent()) {
       if (bodyPart.isLast()) {
         // no need to stream response, we already have it all
+        logger.debug("Single part (size = {}bytes).", bodyPart.getBodyPartBytes().length);
```

Comment:
Grizzly uses the buffer directly instead of asking for the bytes, we should check the log level to avoid getting the bytes otherwise.